src/main/kotlin/net/maizegenetics/phgv2/brapi/model/SampleListResponse.kt

```diff
@@ -12,6 +12,7 @@
 package net.maizegenetics.phgv2.brapi.model
 
 import kotlinx.serialization.Serializable
+import model.Context
```

Comment:
Why this new import but not used?

Reply:
@tcasstevens It is used in the class def:

```
data class SampleListResponse(
    val metadata: Metadata,
    val result: SampleListResponseResult,
    val atcontext: Context? = null
)
```